DigitalLearningSolutions.Data.Tests/TestHelpers/UserTestHelper.cs

```diff
@@ -216,24 +216,51 @@ FROM Candidates
 
         public static void GivenDelegateUserIsInDatabase(DelegateUser user, SqlConnection sqlConnection)
         {
-            sqlConnection.Execute(
-                @"INSERT INTO Candidates (Active, CentreId, LastName, DateRegistered, CandidateNumber, JobGroupID,
-                    Approved, ExternalReg, SelfReg, SkipPW, PublicSkypeLink)
-                  VALUES (@Active, @CentreId, @LastName, @DateRegistered, @CandidateNumber, @JobGroupID,
-                    @Approved, @ExternalReg, @SelfReg, @SkipPW, @PublicSkypeLink);",
+            var userId = sqlConnection.QuerySingle<int>(
+                @"INSERT INTO Users
+                    (
+                        FirstName,
+                        LastName,
+                        PrimaryEmail,
+                        PasswordHash,
+                        Active,
+                        JobGroupID
+                    )
+                    OUTPUT Inserted.ID
+                    VALUES (@FirstName, @LastName, @EmailAddress, @Password, @Active, @JobGroupId)",
                 new
                 {
+                    user.FirstName,
+                    user.LastName,
+                    user.EmailAddress,
+                    user.Password,
                     user.Active,
+                    user.JobGroupId,
+                }
+            );
+            // TODO: UAR - Remove LastName_deprecated from this query once the not-null constraint is lifted

```

Comment:
Good point, I've made a [ticket](https://softwiretech.atlassian.net/browse/HEEDLS-889). Can we point this TODO at that ticket?